app/vmagent/remotewrite/client.go

```diff
@@ -21,11 +20,11 @@ import (
 )
 
 var (
-	sendTimeout = flag.Duration("remoteWrite.sendTimeout", time.Minute, "Timeout for sending a single block of data to -remoteWrite.url")
+	sendTimeout = flagutil.NewArrayDuration("remoteWrite.sendTimeout", "Timeout for sending a single block of data to -remoteWrite.url")
 	proxyURL    = flagutil.NewArray("remoteWrite.proxyURL", "Optional proxy URL for writing data to -remoteWrite.url. Supported proxies: http, https, socks5. "+
 		"Example: -remoteWrite.proxyURL=socks5://proxy:1234")
 
-	tlsInsecureSkipVerify = flag.Bool("remoteWrite.tlsInsecureSkipVerify", false, "Whether to skip tls verification when connecting to -remoteWrite.url")
+	tlsInsecureSkipVerify = flagutil.NewArrayBool("remoteWrite.tlsInsecureSkipVerify", "Whether to skip tls verification when connecting to -remoteWrite.url")
```

Comment:
Make sure that `./vmagent -remoteWrite.tlsInsecureSkipVerify` properly sets the flag to `true` in the same way as `./vmagent -remoteWrite.tlsInsecureSkipVerify=true`. Otherwise this change may break users' setups.

It would be great adding a test for this case in `lib/flagutil`.

Reply:
yep, thought about it, maybe pick first element as default value will work.

Reply:
Added test cases, it already pick first value as default.